modules/olm-installing-operators-from-operatorhub.adoc

```diff
@@ -41,36 +32,20 @@ ifdef::olm-user[]
 
 ifndef::olm-user[]
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-Installation Mode:: Choose *All namespaces on the cluster (default)* to have the
-Operator installed on all namespaces or choose individual namespaces, if
-available, to only install the Operator on selected namespaces. This example
-chooses *All namespaces...* to make the Operator available to all users and
-projects.
+Installation Mode:: Choose *All namespaces on the cluster (default)* to have the Operator installed on all namespaces or choose individual namespaces, if available, to only install the Operator on selected namespaces. This example chooses *All namespaces...* to make the Operator available to all users and projects.
 endif::[]
 ifdef::openshift-dedicated[]
-Installation Mode:: In {product-title} clusters, you can choose *All namespaces on the cluster (default)*
-to have the Operator installed on all namespaces. This makes the Operator
-available to all users and projects.
+Installation Mode:: In {product-title} clusters, you can choose *All namespaces on the cluster (default)* to have the Operator installed on all namespaces. This makes the Operator available to all users and projects.
 endif::[]
 endif::[]
 ifdef::olm-user[]
-Installation Mode:: Choose a specific namespace in which to install the
-Operator.
+Installation Mode:: Choose a specific namespace in which to install the Operator.
 endif::[]
 
-Update Channel:: If an Operator is available through multiple channels, you can
-choose which channel you want to subscribe to. For example, to deploy from the
-*stable* channel, if available, select it from the list.
+Update Channel:: If an Operator is available through multiple channels, you can choose which channel you want to subscribe to. For example, to deploy from the *stable* channel, if available, select it from the list.
 
-Approval Strategy:: You can choose Automatic or Manual updates. If you choose
-Automatic updates for an installed Operator, when a new version of that Operator
-is available, Operator Lifecycle Manager (OLM) automatically upgrades the
-running instance of your Operator without human intervention. If you select
-Manual updates, when a newer version of an Operator is available, OLM creates an
-update request. As a cluster administrator, you must then manually approve that
-update request to have the Operator updated to the new version.
+Approval Strategy:: You can choose Automatic or Manual updates. If you choose Automatic updates for an installed Operator, when a new version of that Operator is available in the selected channel, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select Manual updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
```

Comment:
Actual content change. Specifying "in the selected channel" for Automatic approval strategy.

Reply:
I get why Automatic and Manual are capitalized here, but I don't think it's necessary unless it is matching UI, in which case it should also be in **bold**. IIRC, this is a common convention throughout our docs, so this is just me opining about confusing Capitalization. 👓  ISG does have some general guidance starting on p. 14:
> Use lowercase for names and terms unless the name or term is a proper noun.

Reply:
IIRC some spots did intend to refer to UI elements, but not so much here. Will address.